pom.xml

```diff
@@ -506,7 +506,41 @@
               <propertyExpansion>project.basedir=${project.basedir}</propertyExpansion>
               <sourceDirectory>${project.basedir}</sourceDirectory>
               <includes>**/*</includes>
-              <excludes>src/main/**/*,src/test/java/**/*,src/site/resources/images/**/*,target/**/*,bin/**/*,.git/**/*,.idea/**/*,.settings/**/*,.checkstyle/**/*,.DS_Store/**/*,.nondex/**/*,.pmd/**/*,.fbExcludeFilterFile/**/*,.externalToolBuilders/**/*,.pmdruleset.xml,nbactions.xml,nb-configuration.xml,*.iml</excludes>
+              <!-- All .gitignore files should be added to exclusions -->
```

Comment:
We are not expecting any changes except in StaticVariableName's test and it's inputs.
Please remove all these other changes.